backend/inspirehep/mailing/config.py

```diff
@@ -19,11 +20,12 @@
 MAILTRAIN_URL = "https://lists.labs.inspirehep.net"
 MAILTRAIN_API_TOKEN = ""
 MAILTRAIN_JOBS_WEEKLY_LIST_ID = ""
-SENDGRID_API_KEY = "CHANGE_ME"
-"""SendGrid api authentication key"""
 
 JOBS_DEADLINE_PASSED_SENDER_EMAIL = "jobs@inspirehep.net"
 """Email address for notifying expired jobs"""
 
+MAIL_SERVER = os.environ.get("MAIL_SERVER")
```

Comment:
why? you can already set `INVENIO_MAIL_SERVER` in the env without this line.

Reply:
Because we wanted to be explicit: set INVENIO_MAIL_SERVER var for flask-mail which is not an invenio package, because it is automatically loaded and set in our app, ecc ecc. Too much magic IMO.
But if you guys prefer it, I am okay with it.

Reply:
then I would do
```suggestion
MAIL_SERVER = "localhost"
```
if the goal is to make its existence visible